udata/core/user/api.py

```diff
@@ -251,6 +251,19 @@ def post(self):
         return user, 201
 
 
+@ns.route('/<user:user>/avatar', endpoint='given_user_avatar')
+class UserAvatarAPI(API):
+    @api.secure
+    @api.doc('given_user_avatar')
+    @api.expect(image_parser)
+    @api.marshal_with(uploaded_image_fields)
+    def post(self, user):
+        '''Upload a new avatar for a given user'''
+        parse_uploaded_image(user.avatar)
+        user.save()
```

Comment:
Missing permission check ;-) I think there's a decorator to allow a route only for admins.

Reply:
Done